tests/components/Microsoft.DotNet.UpgradeAssistant.MSBuild.Tests/TargetFrameworkIdentifierTests.cs

```diff
@@ -80,23 +83,23 @@ public void TfmCompare(string target, string tfm, int expected)
             Assert.Equal(expected, result);
 
             static TargetFrameworkMoniker? Create(string? input)
-                => input is null ? null : new TargetFrameworkMoniker(input);
+                => input is null ? null : ParseTfm(input);
         }
 
-        [InlineData("net6.0-windows", "net5.0-windows", "net6.0")]
-        [InlineData("net6.0-windows", "net6.0", "net5.0-windows")]
-        [InlineData("net6.0-windows", "net5.0", "net6.0-windows")]
-        [InlineData("net6.0-windows10.0.5", "net5.0", "net6.0-windows10.0.5")]
-        [InlineData("net6.0-windows10.0.5", "net5.0-windows10.0.5", "net6.0")]
-        [InlineData("net6.0-windows10.1.5", "net5.0-windows10.1.5", "net6.0-windows10.0.5")]
-        [InlineData("net6.0-windows10.1.5", "net5.0-windows10.1.5", "net6.0-windows10.1.5")]
-        [InlineData("net6.0-windows", "net6.0-windows", "net5.0-windows")]
-        [InlineData(null, "net6.0-linux", "net5.0-windows")]
-        [InlineData("net6.0", "net5.0", "net6.0")]
-        [InlineData("net5.0", "net462", "net5.0")]
-        [InlineData("net5.0", "net5.0", "netstandard2.0")]
-        [InlineData("netstandard2.0", "netstandard2.0", "netstandard2.0")]
-        [InlineData("netstandard2.1", "netstandard2.0", "netstandard2.1")]
+        [InlineData(Net60_Windows, Net50_Windows, Net60)]
+        [InlineData(Net60_Windows, Net60, Net50_Windows)]
+        [InlineData(Net60_Windows, Net50, Net60_Windows)]
+        [InlineData(Net60_Windows_10_0_5, Net50, Net60_Windows_10_0_5)]
+        [InlineData(Net60_Windows_10_0_5, Net50_Windows_10_0_5, Net60)]
```

Comment:
Add test scenario `[InlineData(Net60_Windows_10_0_5, Net50_Windows_10_1_5, Net60_Windows_10_1_5)]`

Reply:
That's not the expected result, though. I'd expect `Net60_Windows_10_1_5` as that's what both provided ones are

Reply:
Swapped them around and added the test